### DISCUSSION TITLE
Clarification on beta_channel Usage in MMM Documentation

### DISCUSSION THREAD

**Discussion by myotkh** [pymc-labs/pymc-marketing#1190] [General]
I checked mmm.py and it seems that beta_channel is not used there. However, [this documentation article](https://www.pymc-marketing.io/en/stable/guide/benefits/model_deployment.html) mentions beta_channel, suggesting it should be available as a parameter for setting channel-specific priors.

Could you confirm if beta_channel is intended to be part of the MMM model configuration? If so, could you provide guidance on its implementation? If not, is there an alternative approach, or might this be a documentation inconsistency?

Thank you very much for your assistance!

**Comment by williambdean** [comment]
Hi @myotkh 

The beta is the scaling parameter for many of the saturation transformations. It changed from beta_channel to saturation_beta with the migration to class components. 

If a parameter is in the class but not transformation function, this means that curve asymptotically goes to 1 and we add a multiplication to adjust that. This additional parameter has been named beta. 

Some functions have the asymptote already baked in, like the menten transformation which has alpha instead. 

Hope that clarifies a bit. Let me know if you have any other questions

**Comment by williambdean** [comment]
It looks like that notebook is wrong. Thanks for pointing out!

**Comment by Darren-Li** [comment]
yes, I also find this. Indeed, both in mmm.py and in the website script need to be updated and consistent. Thanks!

**Comment by williambdean** [comment]
Would someone want to make a PR to make the changes? That would be a great help!
@Darren-Li @myotkh
